packages/edit-navigation/src/components/menu-editor/index.js

```diff
@@ -12,16 +18,48 @@ import {
 import { __ } from '@wordpress/i18n';
 import { Panel, PanelBody, Button } from '@wordpress/components';
 import { useViewportMatch } from '@wordpress/compose';
+import { useSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import useNavigationBlocks from './use-navigation-blocks';
 import MenuEditorShortcuts from './shortcuts';
 
-export default function MenuEditor( { menuId, blockEditorSettings } ) {
+export default function MenuEditor( {
+	menuId,
+	blockEditorSettings,
+	onDelete,
+} ) {
+	const kind = 'root';
+	const name = 'menu';
+
 	const [ blocks, setBlocks, saveBlocks ] = useNavigationBlocks( menuId );
 	const isLargeViewport = useViewportMatch( 'medium' );
+	const entities = useSelect( ( select ) =>
+		select( 'core' ).getEntitiesByKind( kind )
+	);
+
+	const entity = find( entities, { kind, name } );
+
+	const deleteMenu = async ( recordId ) => {
+		const path = `${ entity.baseURL + '/' + recordId + '?force=true' }`;
```

Comment:
I think hardcode the base URL and remove the call to `getEntitiesByKind`, since this is all temporary pending a `deleteEntityRecord` action.